R/format_message.R

```diff
@@ -29,8 +29,10 @@ format_message <- function(string, ..., line_length = options()$width) {
     line_length <- 70
   }
 
-  string <- .wrap_message_line(string, line_length)
-  further_lines <- list(...)
+  all_lines <- c(string, ...)
```

Comment:
What's the reason behind this change?

Reply:
Else you can't pass a vector as the first argument 🤷‍♂️